apps/src/netsim/NetSimVisualization.js

```diff
@@ -228,6 +228,9 @@ NetSimVisualization.prototype.setLocalNode = function(newLocalNode) {
     this.eventKeys.remoteChange = newLocalNode.remoteChange.register(
       this.onRemoteChange_.bind(this)
     );
+    this.eventKeys.wireStateChange = newLocalNode.wireStateChange.register(
+      this.onWireStateChange_.bind(this)
+    );
```

Comment:
Make sure you add a corresponding `unregister` operation just above this.

https://github.com/code-dot-org/code-dot-org/blob/686b4d94e195e15972dbc607aeb5eb91463962fd/apps/src/netsim/NetSimVisualization.js#L218-L224